src/vue/backend/src/router/index.js

```diff
@@ -1,22 +1,170 @@
 import Vue from 'vue'
 import VueRouter from 'vue-router'
-import Home from '../views/Home.vue'
 
 Vue.use(VueRouter)
 
 const routes = [
     {
-        path: '/',
-        name: 'Home',
-        component: Home
+        path: '/import/listings',
```

Comment:
We can simplify these import routes. There's a default route for both - the import action, so it's not necessary to give them a specific name, they can just be `''` which is the same as `/`
```
{
    path: '/import',
    children: [
        {
            path: '/listings',
            children: [
                { path: '', name: 'Import IDX Listings' },
                { path: 'Imported', name: 'Imported Listings' }
            ]
        },
        {
            path: '/agents',
            children: [
                // Start w/ default page first.
                { path: '', name: 'Import IDX Agents' },
                { path: 'imported', name: 'Imported Agents' }
            ]
        }
    ]
}
```